### PR TITLE
Fix: arm status [icon]

### PR DESCRIPTION
**Pull request by haslinghuis**
- Loading setup tab is faster then status polling can fill the data so `arming-disable-flags` was not set at first load.
- Armedicon was not working due to CSS bug.
- Motor test was allowed to switch tabs before for testing purposes, now it again picks up status upon re-entering the tab.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by nerdCopter — APPROVED** [review]
* approving. looks  good, but i cannot test the arming icon

**Comment by haslinghuis** [comment]
Rebased

**Comment by haslinghuis** [comment]
Moved the refactoring part to a new PR to ease reviewing.

**Review by nerdCopter — APPROVED** [review]
* approving, tested well. 
* however, i  cannot test arming icon on unsoldered FC.

**Review by McGiverGim — APPROVED** [review]

**Pull request merged**
